src/azul/logging.py

```diff
@@ -163,20 +175,59 @@ def silenced_es_logger():
             assert es_log.level == original_log_level
 
 
-def http_body_log_message(body_type: str,
-                          body: bytes | bytearray | str | IO[bytes] | IO[str] | None,
+type BodyType = Literal['request', 'response']
+
+first_kibibit = 1024
+
+
+def http_body_log_message(body_type: BodyType,
+                          body: bytes | bytearray | str | IO[bytes] | IO[str] | JSON | None,
                           *,
-                          verbatim: bool = False,
+                          debug: int = 1,
                           ) -> str:
+    """
+    Returns a string containing the body_type (request or response) in addition
+    to the actual body (conditional on the debug integer set). The initial KiB
+    of the body is present by defaul (same as debug=1).
+
+    :param body_type: The type of body to specify in the string.
+
+    :param body: The literal body to be used.
+
+    :param debug: The debug level setting the loggin conditions of the body.
+
+    :return: A formatted string containing the body details.
```

Comment:
When the docstring description describes the return value, you don't need to duplicate that here.